src/backend/utils/activity/backend_status.c

```diff
@@ -49,7 +50,6 @@ int			pgstat_track_activity_query_size = 1024;
 /* exposed so that backend_progress.c can access it */
 PgBackendStatus *MyBEEntry = NULL;
 
-
```

Comment:
I know it's probably pgindent's fault, but please remove the whitespace-only changes.